pedpy/methods/density_calculator.py

```diff
@@ -52,13 +52,13 @@ def compute_voronoi_density(
     Args:
         individual_voronoi_data (pd.DataFrame): individual voronoi data per
             frame needs to contain the columns: 'ID', 'frame',
-            'individual voronoi', which holds a shapely.Polygon
+            'voronoi_polygon', which holds a shapely.Polygon
         measurement_area (MeasurementArea): area for which the density is
             computed
     Returns:
-        DataFrame containing the columns: 'frame' and 'voronoi density',
-        DataFrame containing the columns: 'ID', 'frame', 'individual voronoi',
-        'intersecting voronoi'.
+        DataFrame containing the columns: 'frame' and 'density',
+        DataFrame containing the columns: 'ID', 'frame', 'voronoi_polygon',
+        'voronoi_ma_intersection'.
```

Comment:
I suggest to use 'polygon' and 'intersection' as column names.

Further you need to describe the content of the 2nd returned data frame on top. Esp. the intersection of the polygon with the measurement area. Right now nothing is mentioned here.